contracts/RelayHub.sol

```diff
@@ -412,6 +435,24 @@ contract RelayHub is IRelayHub {
         }
     }
 
+    function requiredGas(uint256 relayedCallStipend) public view returns (uint256) {
```

Comment:
There's some light documentation on that method in `IRelayHub`:

```solidity
// Returns how much gas should be forwarded to a call to relayCall, in order to relay a transaction that will spend up to relayedCallStipend gas.
function requiredGas(uint256 relayedCallStipend) public view returns (uint256);
```

Those functions are inside `RelayHub` because they use its private constants (gas overheads, maximum call stipends, etc.).